src/routes/manage_job/get.rs

```diff
@@ -48,6 +61,45 @@ pub struct JobStatusResponseBody {
 	job_status: ValidStatus,
 }
 
+async fn job_result(
+	job_id: i32,
+	req: JobResultRequest,
+	conn_pool: Pool<Postgres>,
+) -> Result<impl warp::Reply, warp::Rejection> {
+	let query = sqlx::query!(
+		r#"
+		SELECT result FROM email_results
+		WHERE job_id = $1
+		ORDER BY id
+		LIMIT $2 OFFSET $3
+		"#,
+		job_id,
+		req.limit as i64,
+		req.offset as i64
+	);
+
+	let rows: Vec<serde_json::Value> = conn_pool
+		.fetch_all(query)
+		.await
+		.map_err(|e| {
+			log::error!(
+				target:"reacher",
+				"Failed to get results for [job_id={}] [limit={}] [offset={}] with [error={}]",
+				job_id,
+				req.limit,
+				req.offset,
+				e
+			);
+
+			ReacherError::from(e)
+		})?
+		.iter()
+		.map(|row| row.get("result"))
+		.collect();
+
+	Ok(warp::reply::json(&rows))
```

Comment:
Can we put the results in a `results` subfield (see the SPEC that I just updated).

The reason is that I'm still thinking whether or not to put pagination metadata in the response too, and this JSON format would allow that (in the future, in a backwards-compatible way)

Reply:
The pagination can certainly be an enhancement for later when it's more likely that end users will consume the API programmatically.

Reply:
Yes, pagination should be done in the future.

I just don't want to break the API, so proposing to only add a `results` subfield for now.

Reply:
I've added the `results` field in the latest commit.